### DISCUSSION TITLE
Welcome to MDsveX Discussions!

### DISCUSSION THREAD

**Discussion by pngwn** [pngwn/MDsveX#214] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ryanatkn** [comment]
hello [Sveltelowda](https://twitter.com/sveltejs/status/1226662581449953280)👐 I'm a Svelte user and a Markdown user and you know, there's an opportunity here. I haven't done it yet, but my plan is to convert the `.md` docs in some projects to MDsveX and add useful stuff like inline interactive examples and entity interlinkages. Once I have something, this seems like a great place to share.

Beyond code docs, I have quite a pile of Markdown documents and it's fun to think about how they can be enhanced with MDsveX. (Foam is an interesting loosely related project) hmm mm
